shell/core/plugins.js

```diff
@@ -110,6 +110,7 @@ export default function({
             this.removePlugin(plugin.name); // Removing this causes the plugin to not load on refresh
 
             // Load all of the types etc from the plugin
+            plugin.builtin = true;
```

Comment:
Only plugins that are included at build time should have the `builtin` flag. With the flag on it'll stop the plugin from being unloaded when logging out.

Think the problem here is that the dashboard is failing to recognise the route is associated with the harvester plugin. It should be doing this, loading the plugin and then allowed navigation to continue.

That code is in `shell/middleware/authenticated.js` and the line that should succeed is `const newLocation = await dynamicPluginLoader.check({ route, store });`